### PR TITLE
Link to docs for changing the build environment in issue templates

### PR DESCRIPTION
**Pull request by jyn514**
Addresses https://github.com/rust-lang/docs.rs/issues/504

The motivation here is to reduce the maintenance burder on the docs.rs team by having people submit PRs to https://github.com/rust-lang/crates-build-env directly. This has 2 main benefits:

1. Crate authors already know the packages they need, so they don't have to tell us and have us make a PR ourselves; this removes an intermediate step.
2. If crate authors are wrong about the packages they need (for instance, https://github.com/rust-lang/docs.rs/issues/483 needed liblapack-dev, not just gfortran), they get immediate feedback instead of having to wait for a deploy and republish their package.

### CODE REVIEW

**Comment by jyn514** [comment]
Note that this only changes the issue templates, not the links on a failed page.

File: README.md [hunk 1]
```diff
@@ -157,6 +157,11 @@ docker exec -it <the container name goes here> psql -U cratesfyi
 docker-compose run -p 3000:3000 web daemon --foreground
 ```
 
+### Changing the build environment
+
+To make a change to https://github.com/rust-lang/crates-build-env
```
Comment (pietroalbini) [inline README.md]: Maybe we want to have nice link texts here? Something like "To make a change to [the build environment] and test that it works on docs.rs, see [the wiki page]"

Comment (jyn514) [reply README.md]: Added this in the next commit.

**Comment by jyn514** [comment]
This is ready for review.

**Comment by pietroalbini** [comment]
Thanks!

**Pull request merged**